ledger/daml-on-sql/src/test/suite/scala/on/sql/CliSpec.scala

```diff
@@ -56,3 +79,7 @@ class CliSpec
   }
 
 }
+
+object CliSpec {
+  private val fakeEnv = mutable.Map[String, String]()
```

Comment:
Please refresh this in a `beforeEach` before each test, instead of hoping that the test will correctly clean it up.

Reply:
Done.

Reply:
Looks good. I'm not a big fan of mutable values in singleton objects; perhaps this can be moved to the class?